docs/practices/2022-05-02 comparers.md

```diff
@@ -0,0 +1,149 @@
+Today's topic is about the standard implementations for the different ways to compare instances of a type. There are 7 interfaces in .NET that are used for this: `IEqualityComparer`, `IEqualityComparer<T>`, `IComparer`, `IComparer<T>`, `Comparable`, `IComparable<T>`, and `IEquatable<T>`. Notice the missing, non-generic IEquatable variant. This is because `object` already has those methods that can be overridden by any derived type. Let's now look at the standard implementations:
```

Comment:
Shall we have a title? 
and maybe change 'Today's topic' as "This post' or "This doc" ...